File: src/main/java/acheng1314/cn/domain/ResponseObj.java
package acheng1314.cn.domain;

public class ResponseObj<T> {
	public final static int OK = 1, FAILED = 0, EMPUTY = -1;
	public final static String OK_STR = "success", FAILED_STR = "failed", EMPUTY_STR = "empty";

	private int code; // 状态码,0成功;1空数据;-1请求失败
	private String msg;
	private Object data;

	public int getCode() {
		return code;
	}

	public void setCode(int code) {
		this.code = code;
	}

	public String getMsg() {
		return msg;
	}

	public void setMsg(String msg) {
		this.msg = msg;
	}

	public Object getData() {
		return data;
	}

	public void setData(Object data) {
		this.data = data;
	}

}

File: src/main/java/acheng1314/cn/domain/ResponseObj.kt
package acheng1314.cn.domain

import java.io.Serializable


open class ResponseObj : Serializable {
    open var code: Int = 0 // 状态码,1成功;-1空数据;0请求失败

    open var msg: String? = null

    open var data: Any? = null

    companion object {
        val OK = 1
        val FAILED = 0
        val EMPUTY = -1
        val OK_STR = "success"
        val FAILED_STR = "failed"
        val EMPUTY_STR = "empty"
    }

    constructor()

    constructor(code: Int, msg: String, data: Any) {
        this.code = code
        this.msg = msg
        this.data = data
    }

}
